Packs/MicrosoftExchangeOnline/Integrations/EWSO365/README.md

```diff
@@ -1,135 +1,167 @@
+| Token / Tenant ID | Token can be received after following the System Integration Setup (Device side steps). | False |
+| Key / Application Secret | Key can be received after following the System Integration Setup (Device side steps). | False |
+| Azure Cloud | Azure Cloud environment. Options are: _Worldwide_ (The publicly accessible Azure Cloud), _US GCC_ (Azure cloud for the USA Government Cloud Community), _US GCC-High_ (Azure cloud for the USA Government Cloud Community High), _DoD_ (Azure cloud for the USA Department of Defense), _Germany_ (Azure cloud for the German Government) | False|
+| Email Address | Mailbox to run commands on and to fetch incidents from. To use this functionality, your account must have impersonation rights or delegation for the account specified. For more information, see https://xsoar.pan.dev/docs/reference/integrations/ewso365\#additional-information | True |
+| UPN Address | When provided, the target mailbox if it's different from the Email Address. Otherwise, the Email Address is used. | False |
+| Name of the folder from which to fetch incidents | Supports Exchange Folder ID and sub-folders e.g. Inbox/Phishing. | True |
+| Access Type | Run the commands using `Delegate` or `Impersonation` access types. | False |
+| Public Folder |  Whether the folder to be fetched from is public. Public folders can store and organize emails on specific topics or projects. Public folders are usually listed under the "Public Folders" section in the navigation pane in the product itself. | False |
+| Fetch incidents |  | False |
+| Incident type |  | False |
+| First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) |  | False |
+| Maximum number of incidents per fetch (up to 200). Performance might be affected by a value higher than 50. |  | False |
+| Mark fetched emails as read | Mark fetched emails as read. | False |
+| Timeout (in seconds) for HTTP requests to Exchange Server |  | False |
+| Trust any certificate (not secure) |  | False |
+| Use system proxy settings |  | False |
+| Run as a separate process (protects against memory depletion) |  | False |
+| Use a self deployed Azure Application | Select this checkbox if you are using a self-deployed Azure application. | False |
+| Incidents Fetch Interval |  | False |
+| What time field should we filter incidents by? | Default is to filter by received-time, which works well if the folder is an "Inbox". But for a folder emails are dragged into for attention, if we filter by received-time, out-of-order processing of emails means some are ignored. Filtering by modified-time works better for such a scenario. This works best if any modifications \(such as tagging\) happens before moving the email into the folder, such that the move into the folder is the last modification, and triggers Cortex XSOAR to fetch it as an incident. | False |
+
+5. Click **Test** to validate the URLs, token, and connection.
+
+## Fetch Incidents
+
+#### Description
```

Comment:
No need for this heading.  Text should be directly under Fetch Incidents heading.